Makefile

```diff
@@ -100,11 +100,6 @@ lint: ## Run gofmt + golangci lint checks
 	$(eval MALFORMED_FILENAMES := $(shell find . | grep -e ':'))
 	@bash -c "[[ '$(MALFORMED_FILENAMES)' == '' ]] || (printf '\nfound unsupported filename characters:\n$(MALFORMED_FILENAMES)\n\n' && false)"
 
-.PHONY: validate-schema
-validate-schema:
-	# ensure the codebase is only referencing a single grype-db schema version, multiple is not allowed
-	python test/validate_schema.py
```

Comment:
@wagoodman besides the simple search and replace, this is part that got removed. There is no need to check if we are importing multiple schemas. Wanted to make sure this was visible to you before merging

Reply:
yup yup, that's correct 👍